src/sentry/deletions/defaults/group.py

```diff
@@ -128,6 +131,17 @@ def delete_bulk(self, instance_list):
 
         group_ids = [group.id for group in instance_list]
 
+        # Tell seer to delete grouping records with these group hashes
+        if group_ids:
+            group = Group.objects.get(id=group_ids[0])
+            project = group.project if group else None
+        if project and features.has("projects:similarity-embeddings-delete-by-hash", project):
+            # TODO (jangjodi): once we store seer grouping info in GroupHash, we should filter by that here
+            group_hashes = GroupHash.objects.filter(project_id=project.id, group__id__in=group_ids)
+            delete_seer_grouping_records_by_hash.apply_async(
+                args=[project.id, [group_hash.hash for group_hash in group_hashes], 0]
+            )
```

Comment:
Call the delete by hash endpoint. This code is called by all group deletion, including projects - still figuring out if there's a way to skip it if it's from project deletion